playgrounds/vue-bun/README.md

```
@@ -0,0 +1,22 @@
+```bash
+pnpm install
+pnpm dev
+```
+
+Open [http://localhost:2999](http://localhost:2999) with your browser to see the result.
+
+## Sponsors
+
+<p align="center">
+  <a href="https://cdn.jsdelivr.net/gh/unnoq/unnoq/sponsors.svg">
+    <img src='https://cdn.jsdelivr.net/gh/unnoq/unnoq/sponsors.svg'/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)